src/components/structures/MessagePanel.js

```diff
@@ -86,6 +86,10 @@ module.exports = React.createClass({
         // to manage its animations
         this._readReceiptMap = {};
 
+        // Remember the read marker ghost node so we can do the cleanup that
+        // Velocity requires
+        this.readMarkerGhostNode = null;
```

Comment:
would be nice to give this a "_" prefix to mark it out as a private member. I can't remember why the other two don't have this, but it's probably historical